### PR TITLE
Updated the session/* articles for Symfony 4

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

File: session.rst [hunk 1]
```diff
@@ -3,6 +3,11 @@ Sessions
 
 .. toctree::
     :maxdepth: 1
-    :glob:
 
-    session/*
+    session/sessions_directory
+    session/avoid_session_start
```
Comment (weaverryan) [inline session.rst]: @sroze not sure if you're the best person, but I was wondering if the above article needs to change at all for the new lazy session handling stuff?

File: session.rst [hunk 2]
```diff
@@ -3,6 +3,11 @@ Sessions
...
+    session/locale_sticky_session
+    session/limit_metadata_writes
+    session/php_bridge
+    session/proxy_examples
```
Comment (weaverryan) [inline session.rst]: @javiereguiluz could we also link to `doctrine/pdo_session_storage` in here without creating issues? I actually think it should be the first or second article.

**Comment by weaverryan** [comment]
Status: Needs work

**Review by weaverryan — CHANGES_REQUESTED** [review]

**Review by wouterj — APPROVED** [review]

File: session.rst (hunk 1 above)
Comment (sroze) [reply session.rst]: Erm, that's a good point. I'll have a look.

**Pull request merged**